### DISCUSSION TITLE
Problems with OpenIA template + Whatsapp

### DISCUSSION THREAD

**Discussion by ghost** [AntlerVC/firetable#1317] [Support / Q&A] [answered]
I have done all the steps needed to configure the template, but when I was in the last step I found some errors installing two extensions called "Twilio whatsapp responder, and Twilio autoreply" but now I solve this problem, but the another problem is the OpenIA isn't generating response for any question.
![Untitled](https://github.com/rowyio/rowy/assets/63557590/545a1930-0f19-42bc-9b76-4320c257f7cc)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @AugustoEstevaoMonte, could you check if you've stored your OpenAI API Key in the [GCP Secret Manager](https://console.cloud.google.com/security/secret-manager) with the name `openai`?
